pkg/azure/ocpgwdeployer.go

```diff
@@ -125,7 +125,12 @@ func (d *ocpGatewayDeployer) Deploy(input api.GatewayDeployInput, status reporte
 	}
 
 	if d.dedicatedGWNode {
-		err = d.deployDedicatedGWNode(machineSets, gatewayNodesToDeploy, status)
+		image, imageErr := d.msDeployer.GetWorkerNodeImage(nil, nil, d.InfraID)
+		if imageErr != nil {
```

Comment:
```suggestion
		image, err := d.msDeployer.GetWorkerNodeImage(nil, nil, d.InfraID)
		if err != nil {
```

Reply:
used a different name since line  135  uses the err variable(declared outside this block) and it's status is checked outside this block at line 138.

Reply:
Why does it matter though?

You return at next line, so it will never reach any code outside the blog.

Not that it matters that much, but you can use `err` here which is more straightforward.

Reply:
yeah I agree with `err` here - it's local and doesn't escape the scope of this block. Also it needs to match L130 below which it doesn't.